specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2017-07-01/bms.json

```diff
@@ -0,0 +1,3568 @@
+          "description": "Additional information about the job."
+        }
+      }
+    },
+    "AzureWorkloadJobExtendedInfo": {
+      "description": "Azure VM workload-specific additional information for job.",
+      "type": "object",
+      "properties": {
+        "tasksList": {
+          "description": "List of tasks for this job",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureWorkloadJobTaskDetails"
+          }
+        },
+        "propertyBag": {
+          "description": "Job properties.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "dynamicErrorMessage": {
+          "description": "Non localized error message on job execution.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureWorkloadJobTaskDetails": {
+      "description": "Azure VM workload specific job task details.",
+      "type": "object",
+      "properties": {
+        "taskId": {
+          "description": "The task display name.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The status.",
+          "type": "string"
+        }
+      }
+    },
+    "BackupManagementUsage": {
+      "description": "Backup management usages of a vault.",
+      "type": "object",
+      "properties": {
+        "unit": {
+          "description": "Unit of the usage.",
+          "enum": [
+            "Count",
+            "Bytes",
+            "Seconds",
+            "Percent",
+            "CountPerSecond",
+            "BytesPerSecond"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "UsagesUnit",
+            "modelAsString": true
+          }
+        },
+        "quotaPeriod": {
+          "description": "Quota period of usage.",
+          "type": "string"
+        },
+        "nextResetTime": {
+          "format": "date-time",
+          "description": "Next reset time of usage.",
+          "type": "string"
+        },
+        "currentValue": {
+          "format": "int64",
+          "description": "Current value of usage.",
+          "type": "integer"
+        },
+        "limit": {
+          "format": "int64",
+          "description": "Limit of usage.",
+          "type": "integer"
+        },
+        "name": {
+          "$ref": "#/definitions/NameInfo",
+          "description": "Name of usage."
+        }
+      }
+    },
+    "BackupManagementUsageList": {
+      "description": "Backup management usage for vault.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of backup management usages for the given vault.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BackupManagementUsage"
+          }
+        }
+      }
+    },
+    "BackupStatusRequest": {
+      "description": "BackupStatus request.",
```

Comment:
Please add description to properties.

Reply:
For all missing descriptions for properties, we will take them in next iteration as this iteration is targeted for validation errors mainly for ARM compliance.